src/async-tokenize/async-tokenize.ts

```diff
@@ -5,7 +5,7 @@ import {
   removeFormElements,
   createFormElement,
   getFormValues,
-} from 'src/common/utils'
+} from '../common/utils'
```

Comment:
Eu subi um PR na main que corrige o problema de o teste não conseguir parsear os caminhos absolutos, podemos voltar como antes:

```suggestion
} from 'src/common/utils'
```